lib/experimental/new_embed.dart

```diff
@@ -171,6 +177,19 @@ class NewEmbed {
       });
     }
 
+    tabController.setTabVisibility('test', false);
+    showTestCodeCheckbox = DInput(querySelector('#show-test-checkbox'));
+    showTestCodeCheckbox.onClick.listen((e) {
+      _showTestCode = !_showTestCode;
+      tabController.setTabVisibility('test', _showTestCode);
+    });
+
+    morePopover = DElement(querySelector('#more-popover'));
+    moreButton = DisableableButton(querySelector('#more-button'), () {
+      _popoverHidden = !_popoverHidden;
```

Comment:
Is this planned for use elsewhere (`_popoverHidden`)? If not, could you ditch it and just toggle the class based on whether it's currently present in morePopover's list of classes?